### ISSUE TITLE
focus outline doesn't have enough contrast

### ISSUE THREAD

**Issue by mfranzke** [db-ux-design-system/db-ux-design-system.github.io#19]
The blue keyboard focus outline doesn't have enough contrast (needs to be 3:1) against it's red background:

<img width="327" height="181" alt="Image" src="https://github.com/user-attachments/assets/3d4bc6ee-4924-4e4b-af83-8397dda73edb" />

https://design-system.deutschebahn.com/
